node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -195,7 +196,8 @@ func (r OsDeviceConnectivityHelperScsiGeneric) GetMpathDevice(volumeId string) (
 
 func (r OsDeviceConnectivityHelperScsiGeneric) flushDeviceBuffers(deviceName string) error {
 	devicePath := filepath.Join(DevPath, deviceName)
-	_, err := r.Executer.ExecuteWithTimeout(TimeOutBlockDevCmd, blockDevCmd, []string{"--flushbufs", devicePath})
+	logger.Debugf("executing command : {--flushbufs} on device : {%v}.", deviceName)
```

Comment:
it's not clear from the message that the command in use is `blockdev`, nor its timeout.
is this to avoid the empty `Output from command: ` message? if so, we could just avoid it somehow inside `ExecuteWithTimeout` when the output is empty

Reply:
no, its to avoid putting timeout and anything that is not interesting

Reply:
why do we need the time out in the node?
plus, `ExecuteWithTimeout ` can be called in a for loop and the log gets bigger in every call so even if part of the log there contains what I added I still want it out of there and out of the loop altogether.

Reply:
the timeout is useful info (e.g. when encountering long executions), same as `deviceDeletePath`, timestamps, etc...

I don't see a loop in `flushDeviceBuffers` nor in `ExecuteWithTimeout`, so the change in the line which this conversation is about, will not take the `executing command` message out of any loop

Reply:
specifically, here the whole function was called in a loop so I didn't have a way to put the command out of it, and I don't think the calling function (that calls to `flushDeviceBuffers`) should know about the args of the command inside..

Reply:
I don't understand your example for useful info,

Reply:
e.g. if any of our timeouts is not quite calibrated, and the relevant timeout value is included with the messages around the time of the issue, it would be easier to see spot it

Reply:
I added timeout

Reply:
I don't see the added timeout - link to line?

Reply:
didnt commit, now its in